podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -8,31 +7,53 @@
 #
 FROM registry.access.redhat.com/ubi9/ubi:9.2
 
-ARG GO_VERSION="1.20.10"
-ARG PROTOC_VERSION="3.11.4"
+ARG ARCH="s390x"
+ARG PACKER_VERSION="v1.9.4"
+ARG PLATFORM="Linux"
+ARG GO_VERSION="1.21.3"
+ARG PROTOC_VERSION="24.3"
+ARG PROTOC_ARCH="s390_64"
```

Comment:
I suppose we don't want this to be the default

Reply:
agree